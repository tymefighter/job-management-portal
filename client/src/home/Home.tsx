import HomeItem from "./HomeItem";

import "../styles/Home.scss";
import "../styles/HomeItem.scss";

function Home() {
    return (
        <div className="home">
            <HomeItem
                name="Job Management"
                imageUrl="images/job-management.jpg"
            >
                Manage all your jobs at one place with our Job Management Application.
                This application allows you to add, delete and edit companies and jobs
                as you like, while providing you with an amazing user experience.
            </HomeItem>
            <HomeItem
                className="home-item--alternate"
                name="Manage Many Jobs "
                imageUrl="images/multiple-jobs.jpeg"
            >
                This application allows you to keep information about many jobs secure
                at one place. This helps you to view all relevant information at your 
                convenience. This helps you improve upon your job archive and hence 
                your ability to pursue them.
            </HomeItem>
            <HomeItem
                name="Manage Many Companies"
                imageUrl="images/multiple-companies.jpeg"
            >
                Manage all the companies and their job posting at this one place.
                This helps you optimize your information retrieval and storage,
                and hence helps you focus on important things such as competitor
                companies.
            </HomeItem>
        </div>
    );
}

export default Home;