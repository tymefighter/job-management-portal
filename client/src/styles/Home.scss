.home {
    display: flex;
    flex-direction: column;
    gap: 2rem;
    margin: 0 2rem;
}