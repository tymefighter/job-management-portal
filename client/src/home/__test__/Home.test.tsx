import { render } from "@testing-library/react";
import Home from "../Home";

test("Home snapshot test", () => {
    const {container} = render(<Home />);

    expect(container.firstChild).toMatchSnapshot("Home Component Snapshot");
});