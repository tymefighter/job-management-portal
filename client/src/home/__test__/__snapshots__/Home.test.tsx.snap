// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test: Home Component Snapshot 1`] = `
<div
  class="home"
>
  <div
    class="home-item"
  >
    <img
      alt="Job Management"
      class="home-item__image"
      src="images/job-management.jpg"
    />
    <h2
      class="home-item__name"
    >
      Job Management
    </h2>
    <p
      class="home-item__desc"
    >
      Manage all your jobs at one place with our Job Management Application. This application allows you to add, delete and edit companies and jobs as you like, while providing you with an amazing user experience.
    </p>
  </div>
  <div
    class="home-item home-item--alternate"
  >
    <img
      alt="Manage Many Jobs "
      class="home-item__image"
      src="images/multiple-jobs.jpeg"
    />
    <h2
      class="home-item__name"
    >
      Manage Many Jobs 
    </h2>
    <p
      class="home-item__desc"
    >
      This application allows you to keep information about many jobs secure at one place. This helps you to view all relevant information at your convenience. This helps you improve upon your job archive and hence your ability to pursue them.
    </p>
  </div>
  <div
    class="home-item"
  >
    <img
      alt="Manage Many Companies"
      class="home-item__image"
      src="images/multiple-companies.jpeg"
    />
    <h2
      class="home-item__name"
    >
      Manage Many Companies
    </h2>
    <p
      class="home-item__desc"
    >
      Manage all the companies and their job posting at this one place. This helps you optimize your information retrieval and storage, and hence helps you focus on important things such as competitor companies.
    </p>
  </div>
</div>
`;
